project_wip_timesheet/__init__.py

```diff
@@ -0,0 +1,4 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
```

Comment:
Codacy found an issue: [Missing docstring in public package (D104)](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6119862)